packages/shared-ui/src/i18n/en.json

```diff
@@ -21,12 +21,12 @@
         "header": "Check your inbox to create your account",
         "body_text1": "We just sent you a magic link to ",
         "body_text2": ". Click on the link in the email to finish setting up your account.",
+        "button_sendLinkAgainWaitingText": "Send email again ({{remainingTime}})",
```

Comment:
please add a (sec) at the end here
<img width="315" alt="image" src="https://github.com/corbado/javascript/assets/134682658/33edac64-cc64-4ce5-9442-f54abee0ac50">